.github/scripts/cluster_creation_1.22-us-east-2.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: eksup-cluster-1-22-us-east-2
+  region: us-east-2
+  version: "1.22"
+
+managedNodeGroups:
```

Comment:
This is a great start but its also somewhat starting in the "ideal" scenario. We should look at purposely starting from a state that is far from ideal to ensure that the process will upgrade correctly across the known hurdles we throw its way. Here is an example "non-ideal" cluster in Terraform that I have used for executing checks on https://github.com/clowdhaus/eksup/blob/7980b93818f55b5c3d795dd1cc430832dafd09e8/examples/mixed/main.tf#L53-L155
- Control plane does not match data plane, there are various levels of skew
- Addons are intentionally out of date and even not supported on next Kubernetes version
- Mix of managed, self-managed, and Fargate profiles

Reply:
Note: Just to be clear, I am not proposing Terraform here - its more of the fact that we start with a cluster that would have several hurdles that it would need to overcome in the upgrade process. Make sure it upgrades the managed nodegorup first if it has a skew of +1 already since thats required by EKS, make the VPC CNI upgrade several times in sequence across one minor version at a time, etc.

Reply:
Thank you. Agreed on the required breadth on these tests.  Will take a look at some of those examples and have those in our eksctl configs as well.